server/src/cores/system-config.core.ts

```diff
@@ -169,16 +169,20 @@ let instance: SystemConfigCore | null;
 
 @Injectable()
 export class SystemConfigCore {
-  private logger = new ImmichLogger(SystemConfigCore.name);
   private configCache: SystemConfigEntity<SystemConfigValue>[] | null = null;
 
   public config$ = new Subject<SystemConfig>();
 
-  private constructor(private repository: ISystemConfigRepository) {}
+  private constructor(
+    private repository: ISystemConfigRepository,
+    @Inject(ILoggerRepository) private logger: ILoggerRepository,
```

Comment:
Here the same thing. You don't want to get something injected when you call the constructor yourself (using the create method)